extensions/amp-script/amp-script.md

```diff
@@ -23,37 +23,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# amp-script
+The `amp-script` component allows you to run custom JavaScript. To maintain AMP's performance guarantees, your code runs in a Web Worker, and certain restrictions apply.
 
-## Overview
-
-The `amp-script` component allows you run custom JavaScript to render UI elements, such as a React component.
-
-### A simple example
+## Usage
 
 An `amp-script` element can load JavaScript in two ways:
 
-- Remotely, from a URL to a JavaScript file.
-- Locally, from a `script[type=text/plain][target=amp-script]` element on the page.
+- remotely, from a URL
+- locally, from a `<script>` element on the page
 
-#### Load JavaScript from a remote URL
+#### From a remote URL
```

Comment:
I'd prefer to keep this as the longer title so that developers can skim the TOC with fuller understanding

Reply:
makes sense!